src/libraries/Common/src/System/Composition/Diagnostics/DebuggerTraceWriter.cs

```diff
@@ -10,7 +10,7 @@ namespace System.Composition.Diagnostics
 {
     internal sealed class DebuggerTraceWriter : TraceWriter
     {
-        private static readonly string s_sourceName = typeof(DebuggerTraceWriter).Assembly.GetName().Name;
+        private const string s_sourceName = "System.ComponentModel.Composition";
```

Comment:
This file is in the `Common` path, I don't think this value will always be `System.ComponentModel.Composition`.

https://github.com/dotnet/runtime/search?q=%22DebuggerTraceWriter.cs+%22&type=

Reply:
Sounds better to revert back?  CC @stephentoub

Reply:
Hmm.  Ok.